apps/datahub/src/app/record/header-record/header-record.component.html

```diff
@@ -44,7 +53,19 @@
       [style.color]="foregroundColor"
     >
       <gn-ui-badge
-        *ngIf="(isGeodata$ | async) === true"
+        [style.--gn-ui-badge-padding]="'2px 8px'"
+        [style.--gn-ui-badge-background-color]="'transparent'"
+        [style.--gn-ui-badge-opacity]="'1'"
+        class="leading-[1.4em] border border-white rounded-md"
+        *ngIf="metadata?.kind"
```

Comment:
What about having a component dedicated to the badge type? because this display is used in record page, also in record cards, and also in the (new) filter on type.
But maybe it can be done in another PR.